src/Languages/ru-RU.xml

```diff
@@ -82,6 +82,7 @@
     <DeleteCurrentLine>Удалить текущую строку</DeleteCurrentLine>
     <Width>Ширина</Width>
     <Height>Высота</Height>
+    <Collapse>Collapse</Collapse>

```

Comment:
Is this missing? (context menu in shortcuts)
![image](https://user-images.githubusercontent.com/1261331/75701072-644eab00-5cb3-11ea-9bc2-b856a3f256f6.png)

Reply:
Yeah, you're right.